### PR TITLE
Feat timer fix

### LINKED ISSUE #50: Timer bug
image

### PR DESCRIPTION
**Pull request by cursorweb**
This fixes #50 

uses `moment-timezone` to do a very elegant solution in my opinion

it was extremely hard and very time consuming but I did it!

### CODE REVIEW

**Review by mike-weiner — COMMENTED** [review]
Nice work! These libraries like `moment.js` provide so many nice convenience functions. 

One thing I did notice is that the page content is no longer centered. (See screenshot below). Is this intended behavior? I don't see any code changes to the CSS would have this effect.

Screenshot-2022-12-16-06 32 03PM

File: components/Leaderboard/leaderboard.module.scss [hunk 1]
```diff
@@ -28,7 +28,9 @@
     }
 
     .section {
-        margin-top: 1.5vh;
+        position: relative;
```
Comment (mike-weiner) [inline components/Leaderboard/leaderboard.module.scss:31]: This might be why the page contents are no longer centered.

Comment (cursorweb) [reply components/Leaderboard/leaderboard.module.scss:31]: oh i didn't notice that oops

Comment (cursorweb) [reply components/Leaderboard/leaderboard.module.scss]: okay it turned out to be a little bit below that, the width: 100%; was messing with flex

**Review by mike-weiner — APPROVED** [review]
Changes look good and things work correctly on my local machine.

**Pull request merged**